README.md

```diff
@@ -16,7 +16,7 @@ Xcode doesn't behave as expected.
 ## Requirements
 
 * OS X Yosemite or El Capitan
-* Xcode 7.0, 7.1, 7.2, 7.3
+* Xcode 7.0, 7.1, 7.2, 7.3, 8.0
```

Comment:
Please add note & link to https://github.com/alcatraz/Alcatraz/issues/475 that there's workaround for Xcode 8.0 and that it doesn't work without it. Thanks. Then I'll merge it asap.